Earthfile

```diff
@@ -206,8 +214,9 @@ test-adapters:
     WITH DOCKER --pull docker.redpanda.com/vectorized/redpanda:v23.2.3
         RUN --mount=$EARTHLY_RUST_CARGO_HOME_CACHE --mount=$EARTHLY_RUST_TARGET_CACHE docker run -p 9092:9092 --rm -itd docker.redpanda.com/vectorized/redpanda:v23.2.3 \
             redpanda start --smp 2  && \
+            (google-cloud-sdk/bin/gcloud beta emulators pubsub start --project=feldera-test --host-port=127.0.0.1:8685 &) && \
             sleep 5 && \
-            RUST_BACKTRACE=1 cargo test --package dbsp_adapters --package sqllib
+            RUST_BACKTRACE=1 cargo test --package dbsp_adapters --features "pubsub-emulator-test" --package sqllib
```

Comment:
can we make this feature the default so we can just continue to invoke cargo test rather than having to remember which features need to be enabled for it?

Reply:
then you'll need to start the pubsub emulator in addition to Kafka to run cargo test. The idea was that most changes will not affect this connector, and if they do, CI will catch it.

Reply:
I see I figured the emulator was embedded inside the test binary